### ISSUE TITLE
Won't install with Laravel 5.3.4

### ISSUE THREAD

**Issue by 2Foot** [YABhq/Laracogs#77]
I just created a new laravel project and it's version 5.3.4, trying to install laracogs results in a failure. Here is what happens:

Using version ^2.0 for yab/laracogs
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Installation request for yab/laracogs ^2.0 -> satisfiable by yab/laracogs[v2.0.0].
    - Conclusion: remove laravel/framework v5.3.4
    - Conclusion: don't install laravel/framework v5.3.4
    - yab/laracogs v2.0.0 requires illuminate/support 5.3 -> satisfiable by laravel/framework[v5.3.0], illuminate/support[v5.3.0].
    - Can only install one of: laravel/framework[v5.3.0, v5.3.4].
    - don't install illuminate/support v5.3.0|don't install laravel/framework v5.3.4
    - Installation request for laravel/framework (locked at v5.3.4, required as 5.3.*) -> satisfiable by laravel/framework[v5.3.4].

Installation failed, reverting ./composer.json to its original content.

**Comment by mlantz** [comment]
Waiting on my buddies with LaravelCollective to do a release that works with 5.3 but thank for putting up an issue.

**Comment by 2Foot** [comment]
Thanks for the update. I just didn't expect the failure as the site indicates 5.3+ and makes no mention of this issue that I could find. Thanks for the speedy response.

**Comment by mlantz** [comment]
lol for what it counts I didnt anticipate on the issue. I messaged the guys at LaravelCollective to help move the support forward and they simply said they've got it, should be good to go by Monday ~ish feel free to promote Laracogs to your friends too I'm always looking for new voices in the community :)

**Comment by 2Foot** [comment]
Matt,

Just got around to trying out Laracogs again, everything seemed to go
smooth with the install however, I do get one error when I try to run the
command "php artisan route:cache", it gives me this error you see below:

php artisan vendor:publish --provider="Yab\Laracogs\LaracogsProvider"

when I commented out that route from the api routes file then the command
works. I just don't know what's wrong with it. Additionally, none of the
routes seem to be working for me although they are in the web routes file.
When I go to my local development server at  localhost/myproject/public/ it
will load the basic laravel page but with no login option etc. If i try to
go to one of the other routes such as: localhost/myproject/public/dashboard
it tells me it is not found on this server. I tried the "login" route etc
as well and get the same result. Is there something I am doing wrong? The
routes exist and the files exist. Sorry, still quite new to web
development, linux and laravel so you'll have to excuse my ignorance if
it's something simple. I did go through all the step listed on your site,
one by one and everything went smoothly - if that means anything. Lol.

Thanks,

Craig
